src/views/Sales/index.tsx

```diff
@@ -90,7 +90,7 @@ export function SalesView() {
 
   return (
     <SaleContext.Provider value={{ SaleShow, setSaleShow }}>
-      <AbsoluteContainer minHeight="200%" inner={false} noPadding={true}>
```

Comment:
If this is something intentional please let me know and I will remove it.

Reply:
This was introduced by an old developer. It is indeed an extra component in the tree.

Reply:
I'll keep the component for now since it has some stylings in it. However I'll keep the minHeight removed.